lib/github-glue.ts

```diff
@@ -137,11 +137,11 @@ export class GitHubGlue {
                             throw new Error("Found");
                         }
                     });
-                } catch {
+                } catch(e1) {
```

Comment:
We seem to consistently leave a space between `catch` and the parameter list, to distinguish this from a function call.

Also, I seem to recall that the convention for unused parameters is to prefix them with `_`, or to just use an underscore to begin with.

Reply:
The underscore as a "discard qualifier" works in C#, but not in TS - I tried as I had the same idea.
But I can prefix with underscore of course.

Reply:
> The underscore as a "discard qualifier" works in C#, but not in TS

It seems to work for me, though.

Reply:
When I used it in two places, I got a warning about a re-used variable, so it seems that it's just a variable named `_`.
The semantics of a "discard qualifier" would be that you can use it multiple times without the compiler assuming it to be a named variable.

The warning could also be a linting mistake, but a search for "typescript discard qualifier _" didn't show any hints that the underscore would be supported as such.